lib/server.js

```diff
@@ -210,12 +212,30 @@ class Server {
     });
 
     const port = findPort(serverOptions.port, this.options.port);
+    const tlsCert = serverOptions.tlsCert || this.options.tlsCert;
+    const tlsKey = serverOptions.tlsKey || this.options.tlsKey;
+    const hostname = serverOptions.hostname || this.options.hostname;
+
+    // NOTE: Before hostname support, jasmine-browser-runner would listen on
+    // all IPs (no hostname) and point browsers to "localhost".  We preserve
+    // backward compatibility here by using different defaults for these two
+    // things.
+    const listenOptions = {
```

Comment:
Good call. I think it'd be better to listen only on loopback by default but that's a change for the next major release.